config/initializers/job_configurations.rb

```diff
@@ -32,7 +32,7 @@
       },
       # Send new device alert notifications
       create_new_device_alert_send_emails: {
-        class: 'CreateNewDeviceAlert',
+        class: 'CreateNewDeviceAlertJob',
```

Comment:
I'm not sure we want to be so strict, but if we have a convention for class names ending in "Job" or "Report", we could have a test for that?

(Which, as an aside, would also reveal that [`GrantRequestsAndSendEmails`](https://github.com/18F/identity-idp/blob/main/app/services/account_reset/grant_requests_and_send_emails.rb) has the same problem as what's being addressed here)

Reply:
I had thought about writing a lint or similar for that, but it seemed like more effort than it might be worth. Iterating over the configuration hash is easier. Will try to add something here: https://github.com/18F/identity-idp/blob/7e35b9bf5974a96d08ba408e25393e4a1a1321a8/spec/config/initializers/job_configurations_spec.rb#L5-L12

Reply:
I've renamed `GrantRequestsAndSendEmails` and added a spec for job naming. There are a few report jobs that don't end with `Job` or `Report`, so I've made the name constraint more relaxed to only require it contain `Report` or end with `Job`.